lib/at_cmd/CMakeLists.txt

```diff
@@ -6,5 +6,13 @@
 
 if(CONFIG_AT_CMD)
     zephyr_library()
+    #b_jh
+    if(CONFIG_POSIX_API)
+        zephyr_library_include_directories(${ZEPHYR_BASE}/include/posix)
```

Comment:
I don't think this belongs here. If either iperf/curl/mosh need to have `/include/posix` in the include path, it should rather be put in the respective CMakeLists file.

Reply:
Yeah. I think that all related stuff came in already in: https://github.com/nrfconnect/sdk-nrf/pull/4074

Reply:
Changes removed.